flink-ml-core/src/main/java/org/apache/flink/ml/linalg/DenseMatrix.java

```diff
@@ -0,0 +1,67 @@
+package org.apache.flink.ml.linalg;
```

Comment:
Add the apache licence ?

Reply:
OK, I will add it later.

Reply:
I have added the apache lisence. At the same time, I refine the KnnModelData as POJO format requiring.